integration-tests/debezium-common/pom.xml

```diff
@@ -58,32 +74,36 @@
             <artifactId>quarkus-junit5</artifactId>
             <scope>test</scope>
         </dependency>
-
         <dependency>
             <groupId>io.rest-assured</groupId>
             <artifactId>rest-assured</artifactId>
             <scope>test</scope>
         </dependency>
-
-        <!-- test dependencies - camel-quarkus -->
         <dependency>
             <groupId>org.apache.camel.quarkus</groupId>
             <artifactId>camel-quarkus-integration-testcontainers-support</artifactId>
             <scope>test</scope>
         </dependency>
-
         <dependency>
             <groupId>org.testcontainers</groupId>
             <artifactId>postgresql</artifactId>
             <scope>test</scope>
         </dependency>
-
         <dependency>
             <groupId>org.postgresql</groupId>
             <artifactId>postgresql</artifactId>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>mysql</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>mysql</groupId>
+            <artifactId>mysql-connector-java</artifactId>
```

Comment:
This must be with scope provided, it's not compliant with the ASF license.

Reply:
@oscerd I'm not still 100% sure about the licensing. But I've removed mysql-connector. It seems that it is transitively add also by quarkus-jdbc-mysql

> [INFO] |  \- io.quarkus:quarkus-jdbc-mysql:jar:1.5.0.CR1:compile
> [INFO] |     \- mysql:mysql-connector-java:jar:8.0.20:compile

So I can run mysql container without need of license (eula), but I can not use driver, right?
Does it mean, that it shouldn't be transitive in quarkus-jdbc-mysql, or is this eula linked only to mysql-connector-java? (wich doesn't seem to be correct, because I'm still using it)

Reply:
mysql container doesn't require EULA, mssql server does. Quarkus doesn't apply the ASF license policy or at least it's not subject to the ASF rules, they only use the license. They shouldn't doing it, because mysql-connector-java is not compliant with ASF license, but it's up to  them. In camel-quarkus we should exclude the mysql-connector-java from the quarkus-jdbc-driver and redeclare it as provided. You're mixing mysql and mssql: EULA is related only to the mssql testcontainer container. Mysql-connector-java has a licensing problem related to ASF license compliancy: we cannot use it explicitly in our projects, we need to make users providing it, that's why it should be provided as scope.